File: 2525-categorize-box-according-to-criteria/2525-categorize-box-according-to-criteria.py
class Solution(object):
    def categorizeBox(self, l, w, h, mass):
        if (l*w*h>=10**9 or (l>=10**4 or w>=10**4 or h>=10**4)):
            if mass>=100:
                return "Both"
            return "Bulky"
        elif mass>=100:
            return "Heavy"
        
        return "Neither"

File: 2525-categorize-box-according-to-criteria/2525-categorize-box-according-to-criteria.java
class Solution {
    public String categorizeBox(int length, int width, int height, int mass) {
        long volume=(long)length*width*height;
        if(volume>=1000000000L || (length>=10000 || width>=10000 || height>=10000)){
            if(mass>=100){
                return "Both";
            }
            return "Bulky";
        }else if(mass>=100){
            return "Heavy";
        }else{
            return "Neither";
        }
    }
}